src/main/java/core/basesyntax/strategy/DataConverterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Action;
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String SEPARATION_PLACE = ",";
+    private static final int ACTION_PART = 0;
+    private static final int FRUIT_PART = 0;
+    private static final int QUANTITY_PART = 0;
```

Comment:
The constants FRUIT_PART and QUANTITY_PART should not have the same value as ACTION_PART. Each constant should represent the correct index for its corresponding part in the string array created by the split operation.